react/features/mobile/navigation/routes.js

```diff
@@ -1,15 +1,18 @@
 export const screen = {
-    root: 'Main root',
     welcome: {
-        main: 'Home',
+        help: 'Help',
+        main: 'Welcome',
+        privacy: 'Privacy',
         tabs: {
             recent: 'Recent',
-            calendar: 'Calendar'
-        },
-        settings: 'Settings',
-        terms: 'Terms',
-        privacy: 'Privacy',
-        help: 'Help'
+            calendar: 'Calendar',
+            settings: {
```

Comment:
Let's move this to the top level, since this is a top level screen now.

Reply:
I created a diferent route obj for SettingsNavigationContainer.